### ISSUE TITLE
NuGet package should following project.json conventions

### ISSUE THREAD

**Issue by ljw1004** [mjsabby/RoslynClrHeapAllocationAnalyzer#24]
The NuGet spec now says that analyzers should go in this directory:
```
$/analyzers/{Framework Name}{Version}/{Supported Architecture}/{Supported Programming Language}/{Analyzer}.dll
```
The ClrHeapAllocation NuGet package seems to put it in a different directory: `$/tools/analyzers/analyzer.dll`.

The NuGet package seems to work okay in packages.config projects, but it didn't work in project.json projects (e.g. UWP, .NETCore). I bet this is caused by the directory layout problem.

**Comment by mjsabby** [comment]
As discussed, PR welcome!

**Comment by ljw1004** [comment]
Fixed by PR #27
